commands/init/deploy.ts

```diff
@@ -0,0 +1,27 @@
+import * as dotenv from 'dotenv'
+import { readFileSync } from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { Args, deploySC, WalletClient, ISCData } from '@massalabs/massa-sc-deployer';
+
+dotenv.config();
+
+if (!process.env.WALLET_PRIVATE_KEY || !process.env.JSON_RPC_URL_PUBLIC) {
+    throw new Error('Missing WALLET_PRIVATE_KEY in .env file');
+}
+
+const deployerAccount = await WalletClient.getAccountFromSecretKey(process.env.WALLET_PRIVATE_KEY);
+
+const __filename = fileURLToPath(import.meta.url);
+
+const __dirname = path.dirname(__filename);
+
+await deploySC(
+    process.env.JSON_RPC_URL_PUBLIC,
+    deployerAccount, 
+    [
+        {data: readFileSync(`${__dirname}/build/main.wasm`), coins: 0, args: new Args().addString('Test')} as ISCData
+    ],
+    0,
+    4_200_000_000
+);
```

Comment:
I'm not a fan of removing the account parameter it's really useful I don't see any default value that are acceptable.
I'm ok for the last two parameters optional.
I'm also not a fan of removing `readFileSync` the people using it are developers are they have developed the contracts I think it's preferable to keep a bit of code and so we have a more flexibility and possibilities. We are making a TS file let's use the possibilities of TS otherwise we can just make a JSON that we parse and polymorph in all ways.

Reply:
I get you points, I was thinking of simplifying the life of the developer, for example with the ability to deploy in 2 lines like in https://hardhat.org/hardhat-runner/docs/guides/deploying or https://trufflesuite.com/docs/truffle/how-to/contracts/run-migrations/

Reply:
I personally find it more complicated to understand in the first one because it's too shady. The second one have the require above. I will open an issue on that but I think for a first version we can keep it like that and decide after if that's ok for you.

Reply:
okay